src/shared/hooks/useCollectionQualityLabels.ts

```diff
@@ -8,13 +8,16 @@ import { ToastService } from '../services';
 
 type UseCollectionQualityLabelsTuple = [QualityLabel[], boolean];
 
-export const useCollectionQualityLabels = (): UseCollectionQualityLabelsTuple => {
+export const useCollectionQualityLabels = (enabled: boolean): UseCollectionQualityLabelsTuple => {
```

Comment:
Ik zou hier `enabled` default op true zetten of de prop vermijden en de implementaties omvormen naar optional render-functie props.